### DISCUSSION TITLE
When can v8.0.24 be expected?

### DISCUSSION THREAD

**Discussion by svenhaag** [primefaces/community#85] [PrimeFaces] [answered]
Hi, due to CVE-2022-45688 in org.json/json which seem to be fixed in PF 8.0.24 (https://github.com/primefaces/primefaces/issues/9752) I like to ask when this release can be expected. Thanks

**Comment by Rapster** [comment, marked as answer]
If I understand well what @melloware is saying it seems to be a false positive. Only few classes are being shaded in PF lib, XML is not one of them.  But I guess there is no harm to cherry pick that commit in PRO/ELITE branches (if possible) Up to @mertsincan

**Reply by melloware** [reply]
@Rapster you have it correct.
